### PR TITLE
Put AMR options in group, add verbosity, fix averages

### PR DESCRIPTION
**Pull request by nilsvu**
### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

File: src/ParallelAlgorithms/Amr/Actions/SendAmrDiagnostics.hpp [hunk 1]
```diff
@@ -10,12 +10,16 @@
...
+#include "IO/Logging/Tags.hpp"
 #include "Parallel/AlgorithmExecution.hpp"
 #include "Parallel/GlobalCache.hpp"
+#include "Parallel/Printf.hpp"
 #include "Parallel/Reduction.hpp"
 #include "ParallelAlgorithms/Amr/Actions/Component.hpp"
 #include "ParallelAlgorithms/Amr/Actions/RunAmrDiagnostics.hpp"
+#include "ParallelAlgorithms/Amr/Tags.hpp"
 #include "Utilities/Serialization/PupBoost.hpp"
+#include "IO/Logging/Verbosity.hpp"
```
Comment (kidder) [inline src/ParallelAlgorithms/Amr/Actions/SendAmrDiagnostics.hpp]: alphabet

**Review by kidder — CHANGES_REQUESTED** [review]
I averaged over size_t as I found the doubles annoying to read, but I don't object to the change...

File: src/ParallelAlgorithms/Amr/Actions/SendAmrDiagnostics.hpp (hunk 1 above)
Comment (kidder) [reply src/ParallelAlgorithms/Amr/Actions/SendAmrDiagnostics.hpp]: also there is a failing check...

Comment (nilsvu) [reply src/ParallelAlgorithms/Amr/Actions/SendAmrDiagnostics.hpp]: fixed!

**Review by kidder — DISMISSED** [review]

**Comment by nilsvu** [comment]
Fixed the input file

**Review by kidder — DISMISSED** [review]

**Review by kidder — APPROVED** [review]

**Pull request merged**